system/jlib/jlog.cpp

```diff
@@ -2995,7 +2995,7 @@ class jlib_decl CComponentLogFileCreator : implements IComponentLogFileCreator,
         maxDetail = DefaultDetail;
         name.set(component); //logfile defaults to component name. Change via setName(), setPrefix() and setPostfix()
         extension.set(".log");
-        local = false;
+        local = true;
```

Comment:
Changed the default because in general you do not want logging from remote processes to be included in a component's log.